app/build.gradle

```diff
@@ -216,6 +224,10 @@ dependencies {
     androidTestImplementation 'com.squareup.okhttp3:mockwebserver:3.10.0'
     testImplementation 'com.squareup.okhttp3:mockwebserver:3.10.0'
 
+    // For the initial release of Glean 19, we require consumer applications to
+    // depend on a separate library for unit tests. This will be removed in future releases.
+    testImplementation "org.mozilla.telemetry:glean-forUnitTests:21.3.0"
```

Comment:
Should this be 24.0.0? Same as `ext.moz_components_version`?

Reply:
No, this is correct.  This is the version of Glean that ships with a-c 2.4.0.0.  This is the gotcha I was talking about that the Glean Gradle plugin solves (handling this mapping automatically).  Unfortunately, the Glean Gradle plugin isn't available prior to a-c 27.0.0.